Src/BootCamp.Chapter/BinaryConverter.cs

```diff
@@ -37,20 +37,22 @@ private static bool IsBinary(string binary)
 
         public static string ToBinary(long number)
         {
-            Stack<int> myBinaryStack = new Stack<int>();
+            string binaryWord = "";
```

Comment:
Prefer to use ``StringBuilder`` instead when concatinating a string in a loop. It is much faster (like a hundred times faster)